flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -861,34 +871,41 @@
             </Component>
 
             <Component ID="EICAS_Screens">
-                <UseTemplate Name="ASOBO_LIGHTING_Knob_Potentiometer_Template">
-                    <POTENTIOMETER>92</POTENTIOMETER>
+
+                <!-- ECAM UPPER DISPLAY BRIGHTNESS -->
```

Comment:
This comment should be consistent with the other ones:

`<!-- UPPER ECAM SCREEN -->`